packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -74,11 +74,11 @@ export class OpGroupingManager {
 	}
 
 	public ungroupOp(op: ISequencedDocumentMessage): ISequencedDocumentMessage[] {
-		if (!isGroupContents(op.contents)) {
+		if (!isGroupContents(op.contents as any)) {
 			return [op];
 		}
 
-		const messages = op.contents.contents;
+		const messages: any[] = (op.contents as { contents: any[] }).contents;
 		let fakeCsn = 1;
```

Comment:
Should these be `unknown` instead of `any`?

Reply:
That doesn't compile.